<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tttExplorerMainWindow</class>
 <widget class="QMainWindow" name="tttExplorerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ttt Explorer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QVTKWidget" name="centralQVTKWidget">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSlider" name="timeSlider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="tracking">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1230</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>F&amp;ile</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionExportData"/>
     <addaction name="actionExportMovie"/>
    </widget>
    <addaction name="actionNewDataset"/>
    <addaction name="actionOpenDataset"/>
    <addaction name="actionCloseDataset"/>
    <addaction name="separator"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>Filters</string>
    </property>
    <widget class="QMenu" name="menuSegmentation">
     <property name="title">
      <string>&amp;Segmentation</string>
     </property>
     <addaction name="actionEdgeInference"/>
    </widget>
    <addaction name="actionDeconvolution"/>
    <addaction name="actionDiffusion"/>
    <addaction name="actionFeature_Detection"/>
    <addaction name="actionVertex_Detection"/>
    <addaction name="menuSegmentation"/>
    <addaction name="actionTracking"/>
    <addaction name="actionJointSegmentationAndTracking"/>
    <addaction name="actionMolecular_Estimation"/>
    <addaction name="actionCellDetector"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuLayers">
    <property name="title">
     <string>Layers</string>
    </property>
    <addaction name="actionAddLayer"/>
    <addaction name="actionRemoveLayer"/>
   </widget>
   <widget class="QMenu" name="menuSelect">
    <property name="title">
     <string>Select</string>
    </property>
    <addaction name="actionSelectVertex"/>
    <addaction name="actionSelectEdge"/>
    <addaction name="actionSelectCell"/>
    <addaction name="actionAddVertex"/>
    <addaction name="actionRemoveVertex"/>
    <addaction name="actionAddEdge"/>
    <addaction name="actionRemoveEdge"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Wi&amp;ndow</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuFrame">
    <property name="title">
     <string>Frame</string>
    </property>
    <addaction name="actionAddFrame"/>
    <addaction name="actionRemoveFrame"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuLayers"/>
   <addaction name="menuFrame"/>
   <addaction name="menuView"/>
   <addaction name="menuFilters"/>
   <addaction name="menuSelect"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewDataset"/>
   <addaction name="actionOpenDataset"/>
   <addaction name="actionCloseDataset"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDeconvolution">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Deconvolution</string>
   </property>
  </action>
  <action name="actionDiffusion">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>D&amp;iffusion</string>
   </property>
  </action>
  <action name="actionNewDataset">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/newfile.svg</normaloff>:/Icons/newfile.svg</iconset>
   </property>
   <property name="text">
    <string>New Dataset</string>
   </property>
  </action>
  <action name="actionOpenDataset">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/open.png</normaloff>:/Icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Dataset</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Reset</string>
   </property>
  </action>
  <action name="actionRemove_Frame">
   <property name="text">
    <string>&amp;Remove Frame</string>
   </property>
  </action>
  <action name="actionAddLayer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Add Layer</string>
   </property>
  </action>
  <action name="actionRemoveLayer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Remove Layer</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionFeature_Detection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Feature Detection</string>
   </property>
  </action>
  <action name="actionVertex_Detection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Vertex Detection</string>
   </property>
  </action>
  <action name="actionEdgeInference_2">
   <property name="text">
    <string>Edge Inference</string>
   </property>
  </action>
  <action name="actionEdgeInference">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Edge Inference</string>
   </property>
  </action>
  <action name="actionTracking">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Tracking</string>
   </property>
  </action>
  <action name="actionJointSegmentationAndTracking">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Joint Segmentation and Tracking</string>
   </property>
  </action>
  <action name="actionMolecular_Estimation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Molecular Estimation</string>
   </property>
  </action>
  <action name="actionCellDetector">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Cell Detector</string>
   </property>
  </action>
  <action name="actionSelectVertex">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Select Vertex</string>
   </property>
  </action>
  <action name="actionSelectEdge">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Se&amp;lect Edge</string>
   </property>
  </action>
  <action name="actionSelectCell">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select &amp;Cell</string>
   </property>
  </action>
  <action name="actionAddVertex">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Add Vertex</string>
   </property>
  </action>
  <action name="actionRemoveVertex">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Remove Vertex</string>
   </property>
  </action>
  <action name="actionAddEdge">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add &amp;Edge</string>
   </property>
  </action>
  <action name="actionRemoveEdge">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Re&amp;move Edge</string>
   </property>
  </action>
  <action name="actionExportData">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Export data</string>
   </property>
  </action>
  <action name="actionExportMovie">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export &amp;movie</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCloseDataset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/close.png</normaloff>:/Icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Close Dataset</string>
   </property>
  </action>
  <action name="actionAddFrame">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Frame</string>
   </property>
  </action>
  <action name="actionRemoveFrame">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remove Frame</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
